backend/data/program-credentials/merge-credentials.py

```diff
@@ -45,48 +45,48 @@ class CredentialType(Enum):
     Certificate = 'Certificate'
     # http://credreg.net/ctdl/terms/credentialType#ApprenticeshipCertificate
     # Credential earned through work-based learning and earn-and-learn models that meet standards and are applicable to industry trades and professions.
-    ApprenticeshipCertificate = 'Apprenticeship'
+    ApprenticeshipCertificate = 'ApprenticeshipCertificate'
     # http://credreg.net/ctdl/terms/credentialType#JourneymanCertificate
     # Credential awarded to skilled workers on successful completion of an apprenticeship in industry trades and professions.
-    JourneymanCertificate = 'Journeyman Certificate'
+    JourneymanCertificate = 'JourneymanCertificate'
     # http://credreg.net/ctdl/terms/credentialType#MasterCertificate
     # Credential awarded upon demonstration through apprenticeship of the highest level of skills and performance in industry trades and professions.
-    MasterCertificate = 'Master Certificate'
+    MasterCertificate = 'MasterCertificate'
 
     # http://credreg.net/ctdl/terms/credentialType#Degree
     # Academic credential conferred upon completion of a program or course of study, typically over multiple years at a college or university.
     Degree = 'Degree'
     # http://credreg.net/ctdl/terms/credentialType#AssociateDegree
     # College/university award for students typically completing the first one to two years of post secondary school education.
-    AssociateDegree = 'Associate Degree'
+    AssociateDegree = 'AssociateDegree'
     # http://credreg.net/ctdl/terms/credentialType#BachelorDegree
     # College/university award for students typically completing three to five years of education where course work and activities advance skills beyond those of the first one to two years of college/university study.
-    BachelorDegree = 'Bachelor Degree'
+    BachelorDegree = 'BachelorDegree'
     # http://credreg.net/ctdl/terms/credentialType#MasterDegree
     # Credential awarded for a graduate level course of study where course work and activities advance skills beyond those of the bachelor's degree or its equivalent.
     MasterDegree = 'Masters Degree'
```

Comment:
Should this be `MastersDegree`?

Reply:
Should actually be `MasterDegree`. Will change. Thanks for the catch!